lib/helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Helper
+class Helper
+  def self.init_s3_from_env
+    home = Dir.home
+    file_writer("#{home}/.s3/key", ENV['S3_KEY'], 'w')
```

Comment:
I think it's no need to extract Dir.home to specific variable, you can call it directly two times